import-service/lambdas/importProductsFile.js

```diff
@@ -0,0 +1,58 @@
+import { S3Client, GetObjectCommand, PutObjectCommand } from "@aws-sdk/client-s3";
+import { getSignedUrl } from "@aws-sdk/s3-request-presigner";
+
+let resault = undefined, stCode = 0;
+
+
+export const handler = async event => {
+
+//	console.log(event);
+ 	
+  if (event === undefined || !event.queryStringParameters || !event.queryStringParameters.name || event.queryStringParameters.name === '') {
+        resault = { message: 'Bad request' }, stCode = 400;
+  
+  } else {
+  
+          let fileName = event['queryStringParameters']['name'];
```

Comment:
What is the goal here of using square brackets to access object property? Why don't you use just dots?

Reply:
Hmmm... It is one of the possible way.
But of course it can be so **event.queryStringParameters.name**
And it looks better.

Reply:
**Thank you for your time and Code Review!**